zmq/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/bash
+
```

Comment:
Check the script with [`shellcheck`](https://www.shellcheck.net/).

Reply:
See the sample `build.sh` scripts from [Python](https://github.com/unikraft/static-pie-apps/pull/10) and [Nginx](https://github.com/unikraft/static-pie-apps/blob/master/nginx/build.sh). Use `pushd` / `popd` to change folders. Print messages for different steps.